src/libraries/System.Net.WebSockets.Client/ref/System.Net.WebSockets.Client.cs

```diff
@@ -18,6 +17,7 @@ public sealed partial class ClientWebSocket : System.Net.WebSockets.WebSocket
         public override System.Threading.Tasks.Task CloseAsync(System.Net.WebSockets.WebSocketCloseStatus closeStatus, string? statusDescription, System.Threading.CancellationToken cancellationToken) { throw null; }
         public override System.Threading.Tasks.Task CloseOutputAsync(System.Net.WebSockets.WebSocketCloseStatus closeStatus, string? statusDescription, System.Threading.CancellationToken cancellationToken) { throw null; }
         public System.Threading.Tasks.Task ConnectAsync(System.Uri uri, System.Threading.CancellationToken cancellationToken) { throw null; }
+        public System.Threading.Tasks.Task ConnectAsync(System.Uri uri, System.Net.Http.HttpMessageInvoker invoker, System.Threading.CancellationToken cancellationToken) { throw null; }
```

Comment:
To fix the compile errors you need to add `
    <ProjectReference Include="$(LibrariesProjectRoot)System.Net.Http\ref\System.Net.Http.csproj" />` to `src/libraries/System.Net.WebSockets.Client/ref/System.Net.WebSockets.Client.csproj`.